src/main/webapp/dev/js/common/questionRank.es6

```diff
@@ -220,10 +220,21 @@ function bindRankEvents() {
         });
 }
 
+function removeInvalidFeedbackPathsForRankRecipientQuestions() {
+    $('input[name="questiontype"][value="RANK_RECIPIENTS"]').each(function () {
+        const $form = $(this).closest('form');
+
+        $form.find('[data-recipient-type="NONE"],[data-recipient-type="SELF"]').hide();
```

Comment:
There's no `OWN_TEAM` here?

Reply:
Yes there should be `OWN_TEAM` here. Will correct it.

Reply:
Upon closer look I saw that there's no need for `[data-recipient-type="OWN_TEAM"]` here because that statement was to exclude invalid recipients selected through the common feedback paths button, it doesn't have a path where recipient is `own team`. So `OWN_TEAM` isn't needed here.